### DISCUSSION TITLE
Are the repos still maintained?

### DISCUSSION THREAD

**Discussion by daviddesmet** [elgatosf/elgato#7] [General]
I just wonder if the repositories are still maintained, I'm personally asking about the [Analog Clock](https://github.com/elgatosf/streamdeck-analogclock) one since I see it hasn't been updated for a while and there are some old issues opened in there.

Anyway, I just filled an issue in there regarding some DST changes in Mexico which are causing the clock to display the wrong time.

**Comment by tiptronic** [comment]
Hi @daviddesmet 

thanks for your comment.

And yes, the repos are maintained and updated regularily.

The 'AnalogClock' plugin is/was basically just a sample to show how to draw a clock onto a key in Stream Deck 4.

It shows the clock according to your local timezone, so the time shown is the time reported by your operating system to Stream Deck.

I added a more detailed answer in the [streamdeck-analogclock](https://github.com/elgatosf/streamdeck-analogclock/issues/6#issuecomment-1518096550) repository's issues section.
